frameworks/Crystal/moonshine/server-redis.cr

```diff
@@ -30,6 +29,11 @@ end
 private def randomWorld
   id = rand(1..ID_MAXIMUM)
   num = REDIS.get("world:" + id.to_s)
+
+  if num == Nil
+    num = 7777
+  end
```

Comment:
This seems wrong to me - if you get a null pointer back from this, then there is something wrong with how we are getting the value for `id`, and we should definitely create special cases to deal with that.